src/Workleap.DomainEventPropagation.Telemetry/Workleap.DomainEventPropagation.Telemetry.csproj

```diff
@@ -39,5 +39,8 @@
     <!-- Exposes internal symbols to test projects and mocking libraries -->
     <InternalsVisibleTo Include="Workleap.DomainEventPropagation.Tests,PublicKey=002400000480000094000000060200000024000052534131000400000100010025301ce547647ab5ac9264ade0f9cdc0252796a257095add4791b0232c1def21bb9e0c87d218713f918565b23394362dbcb058e210c853a24ec33e6925ebedf654a0d65efb3828c855ff21eaaa67aeb9b24b81b8baff582a03df6ab04424c7e53cacbfe84d2765ce840389f900c55824d037d2c5b6b330ac0188a06ef6869dba" />
     <InternalsVisibleTo Include="DynamicProxyGenAssembly2, PublicKey=0024000004800000940000000602000000240000525341310004000001000100c547cac37abd99c8db225ef2f6c8a3602f3b3606cc9891605d02baa56104f4cfc0734aa39b93bf7852f7d9266654753cc297e7d2edfe0bac1cdcf9f717241550e0a7b191195b7667bb4f64bcb8e2121380fd1d9d46ad2d92d2d15605093924cceaf74c4861eff62abf69b9291ed0a340e113be11e6a7d3113e92484cf7045cc7" />
+
+    <!-- TODO: Think about alternative solutions to this (friend assemblies pattern at the moment) -->
+    <InternalsVisibleTo Include="Workleap.DomainEventPropagation.Publishing,PublicKey=002400000480000094000000060200000024000052534131000400000100010025301ce547647ab5ac9264ade0f9cdc0252796a257095add4791b0232c1def21bb9e0c87d218713f918565b23394362dbcb058e210c853a24ec33e6925ebedf654a0d65efb3828c855ff21eaaa67aeb9b24b81b8baff582a03df6ab04424c7e53cacbfe84d2765ce840389f900c55824d037d2c5b6b330ac0188a06ef6869dba" />
```

Comment:
This change is unusual, but there are internal types in the telemetry project that are required in the publishing project.

This can change later on if we have a different strategy, but it is necessary for the current state of things.